packages/binding-file/src/file-client.ts

```diff
@@ -72,6 +72,13 @@ export default class FileClient implements ProtocolClient {
         throw new Error("FileClient does not implement unlink");
     }
 
+    /**
+     * @inheritdoc
+     */
+    public async requestThingDescription(uri: string): Promise<Content> {
+        throw new Error("Not implemented");
```

Comment:
Is there a blocker to implement this or is it just a time constraint? I think we can jus read the file pointed by the file URI.

Reply:
I added an implementation now in 3a7c741e48a155b01039b3745db65fedd402755f :) I think there might actually be some more room for simplifying the `FileClient` implementation, though, since the file extension determination is not actually needed when we assume a default Content-Type of `application/json` for `Form`s. However, this would constitute a breaking change.

Reply:
It is good! I would like to keep that auto detection of content type... TD 2.0 might support other formats than the default.